docs/source/notebooks/general/other_nuts_samplers.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<p>Maybe add that these samplers can be significantly faster and also sample on the GPU.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Sure! Added!</p>